research/nips2018/src/server.py

```diff
@@ -78,7 +78,7 @@ class TimeSignature:
 playable_instruments = set(['click', 'bass', 'drums', 'chords', 'stop'])
 
 # Read in the PerformanceRNN model.
-BASE_MODELS_PATH = '/home/psc/pyosc/psc2/magenta_models'
+BASE_MODELS_PATH = '/Users/korymath/Documents/magenta-demos/ai-jam-js'
```

Comment:
maybe we should make this None or something, so it's clear people need to specify their own paths.
```suggestion
BASE_MODELS_PATH = None
```